src/main/java/com/github/webicitybrowser/spec/fetch/imp/FetchEngineImp.java

```diff
@@ -42,8 +43,15 @@ private FetchResponse httpNetworkFetch(FetchParams params) {
 
 	private void fetchResponseHandover(FetchParams params, FetchResponse response) {
 		if (params.consumeBodyAction() != null) {
-			// TODO: Queue a response
+			if(response.body() == null) {
```

Comment:
I don't think this route is speced to occur when the response body is null
Is there a line of the spec I missed?

Reply:
![image](https://github.com/WebicityBrowser/Webicity/assets/76881722/6a0eebfb-e06b-47b5-9838-74d796da2c79)
There is in point 3. that states if body is null queue a fetch task

Reply:
Ah, apparently I didn't read it very well
In that case, you can remove the TODO statement on line 47

Also, replace line 50 with a TODO statement, since in that case the "fully read" step occurs instead of calling consume body according to the spec